src/main/java/com/example/year_percentage_bot/YearPercentageBot.java

```diff
@@ -1,22 +1,115 @@
 import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboard;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
 
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
 public class YearPercentageBot extends TelegramLongPollingBot {
 
+    private static final String START_BOT_REQUEST = "Start this bot";
+    private static final String STOP_BOT_REQUEST = "Stop this bot";
+    private boolean botRunning = false;
+    private Long chatId = null;
+
     @Override
     public void onUpdateReceived(Update update) {
+        if (update.hasMessage() && update.getMessage().hasText()) {
+            Message message = update.getMessage();
+            try {
+                execute(getResponseMessage(message));
+            } catch (TelegramApiException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private SendMessage getResponseMessage(Message message) {
+        switch (message.getText()) {
+            case "/start":
+                return getStartCommandMessage(message);
+            case START_BOT_REQUEST:
+                botRunning = true;
+                chatId = message.getChatId();
+                return getStartBotCommandMessage(message);
+            case STOP_BOT_REQUEST:
+                botRunning = false;
+                chatId = message.getChatId();
+                return getStopBotCommandMessage(message);
+            default:
+                return new SendMessage(String.valueOf(message.getChatId()), ":)");
+        }
+    }
+
+    private SendMessage getStartCommandMessage(Message message) {
+        SendMessage response = new SendMessage();
+        response.setText("This bot can send messages about how many days of the year " +
+                "have already passed (in percentage), every morning at 09:00");
+        response.setChatId(message.getChatId());
+        response.setReplyMarkup(getMainMenu());
+        return response;
+    }
+
+    private SendMessage getStopBotCommandMessage(Message message) {
+        SendMessage response = new SendMessage();
+        response.setText("Bot was stopped");
+        response.setChatId(message.getChatId());
+        return response;
+    }
+
+    private SendMessage getStartBotCommandMessage(Message message) {
+        SendMessage response = new SendMessage();
+        response.setText("Bot was started");
+        response.setChatId(message.getChatId());
+        return response;
+    }
+
+    //@Scheduled(cron = "0 0 9 * * ?")
+    @Scheduled(cron = "* * * * * *")
+    public void sendDailyMessage() {
+        if (!botRunning || chatId == null) return;
```

Comment:
It's okay for now, but please note that this approach will work only for one user.